apps/condo/domains/marketplace/schema/MarketItemPrice.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Generated by `createschema marketplace.MarketItemPrice 'price:Json; marketItem:Relationship:MarketItem:CASCADE; organization:Relationship:Organization:CASCADE;'`
+ */
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/marketplace/access/MarketItemPrice')
+const { PRICE_FIELD } = require('@condo/domains/marketplace/schema/fields/price')
+
+const MarketItemPrice = new GQLListSchema('MarketItemPrice', {
+    schemaDoc: 'MarketItemPrice is set separately from MarketItem and indicates the price of the service.',
+    fields: {
+
+        price: PRICE_FIELD,
+
+        marketItem: {
+            schemaDoc: 'Link to MarketItem',
+            type: 'Relationship',
+            ref: 'MarketItem',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        organization: {
```

Comment:
Looks like this field is not needed here. We may find the organization within `marketItem.organization` field.
Otherwise, we must control that `marketItem.organization === marketItemPrice.organization`.